babbage/model/dimension.py

```diff
@@ -49,6 +49,10 @@ def cardinality_class(self):
                 return 'low'
             return 'tiny'
 
+    @property
+    def datatype(self):
```

Comment:
In adhering to babbage's pretty good coding practices, I think this should be documented. In particular that it may be None? Is it a type or a string description of the type?